### ISSUE TITLE
Make toot.cat more accessible to people with different levels of vision

### ISSUE THREAD

**Issue by ashyon** [TootCat/mastodon#12]
Many aspects of the website are hard to distinguish. Examples include:
- Faded text in the notifications column
- Faded text in empty columns
- The eye button in the top left of images (hard to see for me, too)
- Search bar?

List any others you can think of in the comments, please and thank you.

I'll be working to fix these issues in my spare time.

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by wavebeem** [comment]
This is great. FYI this is the upstream issue https://github.com/tootsuite/mastodon/issues/317

**Comment by ashyon** [comment]
Thanks @wavebeem :)

**Comment by ashyon** [comment]
![2017-04-11_14 30 43](https://cloud.githubusercontent.com/assets/9281956/24924635/7949dc86-1ec3-11e7-9f57-29ea0197c3c6.png)
![2017-04-11_14 30 28](https://cloud.githubusercontent.com/assets/9281956/24924634/7948c5b2-1ec3-11e7-8b6a-b1cdacf2a79e.png)

**Comment by ashyon** [comment]
Of course, the colors can and should be changed for this but it's just an example of how it could look

**Comment by wavebeem** [comment]
that would be very nice! alternatively you could just add a line around the
quoted toot.
